prow/oss/cluster/hook-ghapp.yaml

```
@@ -127,10 +134,10 @@ spec:
           secretName: kubeconfig-build-looker-private
       - name: hmac
         secret:
-          secretName: hmac-token
-      - name: oauth
+          secretName: ghapp-hmac-token
+      - name: github-app-token
         secret:
-          secretName: oauth-token-2
+          secretName: github-app-token
```

This secret name doesn't match the KES resource: https://github.com/GoogleCloudPlatform/oss-test-infra/pull/1178/files
Should be 
```suggestion
          secretName: ghapp-token
```

---

fixed